app/waf/src/models/application.model.ts

```diff
@@ -0,0 +1,51 @@
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class Application extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+  })
+  id?: number;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  name: string;
+
+  @property({
+    type: 'string',
+  })
+  description?: string;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  declaration: string;
+
+  @property({
+    type: 'string',
+  })
+  status?: string;
+
+  @property({
+    type: 'date',
+  })
+  create_time?: string;
+
+  @property({
+    type: 'date',
+  })
+  update_time?: string;
+
+  @property({
+    type: 'number',
+  })
+  policy_id?: number;
```

Comment:
Renaming it to wafpolicy_id? There are several types of 'policy' in BIG-IP. I feel renaming it to wafpolicy_id will avoid naming conflict, and will be consistent with you wafpolicy model.

Reply:
Agree.